### PR TITLE
Updated to Drupal 10. (#1)

### PR DESCRIPTION
**Pull request by AlexSkrypnyk**
PR against `9.x` as there is no `10.x` yet. Happy to update to `10.x` once it is created.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php-versions: ['7.4', '8.1']
```
Comment (AlexSkrypnyk) [inline .github/workflows/ci.yml:16]: D10 requires 8.1 minimum

File: composer.json [hunk 2]
```diff
@@ -40,7 +40,8 @@
             "composer/installers": true,
             "cweagans/composer-patches": true,
             "drupal/core-composer-scaffold": true,
-            "dealerdirect/phpcodesniffer-composer-installer": true
+            "dealerdirect/phpcodesniffer-composer-installer": true,
+            "phpstan/extension-installer": true
```
Comment (AlexSkrypnyk) [inline composer.json:44]: `phpstan/extension-installer` comes from `drupal/core-dev@^10`

Comment (normanlolx) [reply composer.json:44]: We should probably create a 10x branch here

**Comment by webflo** [comment]
Looks, great. Thanks!

**Comment by webflo** [comment]
Created 10.x with the commit from @AlexSkrypnyk.

**Comment by AlexSkrypnyk** [comment]
@webflo @leymannx 
Thank you for your prompt review and actioning this. 🙏 
You've unblock a lot of people trying to push the Drupal 10 contrib updates!

**Pull request closed without merging**